tests/test_sourmash_sketch.py

```diff
@@ -534,6 +554,31 @@ def test_do_sketch_translate_multik_with_protein():
             assert 10 in ksizes
 
 
+def test_do_sketch_translate_multik_with_protein_from_file():
+    with utils.TempDirectory() as location:
+        testdata1 = utils.get_test_data('short.fa')
+
+        file_list = os.path.join(location, "filelist.txt")
+        with open(file_list, 'wt') as fp:
+            print(testdata1, file=fp)
+
+        status, out, err = utils.runscript('sourmash',
+                                           ['sketch', 'translate',
+                                            '-p', 'k=7,k=10,num=500',
+                                            '--from-file', file_list],
+                                           in_directory=location)
+        outfile = os.path.join(location, 'short.fa.sig')
+        assert os.path.exists(outfile)
+
+        with open(outfile, 'rt') as fp:
+            sigdata = fp.read()
+            siglist = list(signature.load_signatures(sigdata))
```

Comment:
any reasoning behind using this syntax vs `load_file_as_signatures`?

Reply:
hah! no, not really, I just wrote these tests before `load_file_as_signatures` was available, and since it's internal test code I'm not as worried about using the public API.